README.md

```diff
@@ -29,6 +29,8 @@ $ ./launch.sh
 
 Once the server is running, you can run the mock API tests for your SDK. See the SDK's documentation for more information.
 
+You can check which scenarios are included in the package by referencing the README in the package. This README includes information on how to run the server as well as descriptions of each scenario.
```

Comment:
I added this text. If you think it looks good, merge it. I approve of your changes.